samples/nrf9160/gps/src/supl_support.h

```diff
@@ -0,0 +1,69 @@
+
```

Comment:
Missing license header and include guard.

Reply:
Ok, added the extern c lines too, it's meant as in internal API so probably not necessary. But aligns with the rest